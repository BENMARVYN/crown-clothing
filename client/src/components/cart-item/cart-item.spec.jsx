import React from "react";
import { shallow } from "enzyme";
import { CartItem } from "./cart-item.component";

let mockItem;
beforeEach(() => {
  mockItem = {
    imageUrl: "https://fakeimage.com/img.png",
    price: 399,
    name: "Fake Item",
    quantity: 1,
  };
});

describe("Testing CartItem", () => {
  it("Should match snapshot", () => {
    expect(shallow(<CartItem item={mockItem} />)).toMatchSnapshot();
  });
});
