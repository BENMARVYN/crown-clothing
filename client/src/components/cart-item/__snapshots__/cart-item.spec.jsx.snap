// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing CartItem Should match snapshot 1`] = `ShallowWrapper {}`;
